wp-graphql-wpml.php

```diff
@@ -718,30 +660,26 @@ function wpgraphqlwpml__filter_graphql_pre_model_data_is_private($unused, $model
     return null;
 }
 
-function wpgraphqlwpml__filter_graphql_connection_ids(array $ids, AbstractConnectionResolver $resolver)
-{
+function wpgraphqlwpml_filter_graphql_connection_ids(array $ids, AbstractConnectionResolver $resolver): array {
     global $wpgraphqlwpml_prev_language;
     global $wpgraphqlwpml_url_filter_off;
 
     $field_name = $resolver->getInfo()->fieldName;
     if ($field_name === 'menus') {
         global $icl_adjust_id_url_filter_off;
         $icl_adjust_id_url_filter_off = true;
-        $args = $resolver->getArgs();
+        $args = $resolver->get_args();
```

Comment:
The `getArgs` method was deprecated by WP GraphQL. I'm not sure if the new option was already available in older versions of the plugin, so it might be something that should be reverted to ensure compatibility with older versions of WPGQL.